Update azure-stack-register.md

BradleyBartlett opened the pull request:
I have pushed a change to the RegisterWithAzure script on the VNext branch that will perform the process of registering the Microsoft.AzureStack resource provider as part of the script. Documentation for registering the RP is no longer necessary.

azurecla commented:
Hi __@felixrieseberg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

squillace commented:
Hey Felix -- we're in a day-long hold pattern with content PRs right now; I'll have a read and send feedback, but this looks great. Likely merging on Wednesday or Thursday. I'll also track down the right person to own (it'll be me at least initially....)

cephalin commented:
Hi Felix. Thanks for your contribution. I think this one is almost ready to go:
- in the metadata "authors" and "ms.service", change it to your GitHub username "felixrieseberg".
- At the end of line 13, change a to an.

felixrieseberg commented:
@cephalin Thanks for taking a look! I included your changes.

Commit:
Update azure-stack-register.md

I have pushed a change to the RegisterWithAzure script on the VNext branch that will perform the process of registering the Microsoft.AzureStack resource provider as part of the script. Documentation for registering the RP is no longer necessary.
